### PR TITLE
feat: set ss58_format when instantiating the sdk

### PR DESCRIPTION
**Pull request by shuffledex**

### CODE REVIEW

File: src/api/entities/__tests__/Account.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ describe('Account class', () => {
     address = 'someAddress';
     key = 'someKey';
 
-    sinon.stub(utilsConversionModule, 'addressToKey').withArgs(address).returns(key);
+    sinon.stub(utilsConversionModule, 'addressToKey').returns(key);
```
Comment (VictorVicente) [inline src/api/entities/__tests__/Account.ts:26]: Shouldn't it be `.withArgs(account, context)`?

Comment (shuffledex) [reply src/api/entities/__tests__/Account.ts:26]: no need to check attrs in this one, the return value will be always the same

**Review by monitz87 — CHANGES_REQUESTED** [review]

**Review by monitz87 — APPROVED** [review]

**Pull request merged**